lib/modules/platform/azure/index.ts

```diff
@@ -990,3 +994,44 @@ export async function deleteLabel(
   const azureApiGit = await azureApi.gitApi();
   await azureApiGit.deletePullRequestLabels(config.repoId, prNumber, label);
 }
+
+function cachePr(pr?: GitPullRequest): void {
+  config.prList ??= [];
+  if (!pr) {
+    return;
+  }
+
+  const azurePr = getRenovatePRFormat(pr);
+  const idx = getCachedPrIdx(azurePr.number);
+  if (is.undefined(idx)) {
+    config.prList.push(azurePr);
+  } else {
+    config.prList[idx] = azurePr;
+  }
+}
+
+function updateCachedPr(prNo: number, pr?: GitPullRequest): void {
+  const idx = getCachedPrIdx(prNo);
+  if (!config.prList || !pr || !idx) {
```

Comment:
What if idx is `0`?

Reply:
it should be is.undefined sry :D  did the same inside cachePr(